ui/radar/radar_config.js

```diff
@@ -2,6 +2,19 @@
 
 UserConfig.registerOptions('radar', {
   options: [
+    {
+      id: 'BRankEnabled',
+      name: {
+        en: 'B-Rank enabled',
+        de: 'B-Rank enabled',
```

Comment:
Can you remove translation lines for other languages? Other people will come by and find it, but only if it's missing.

Reply:
will do